### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kushal34712**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by honey-speaks-tech — COMMENTED** [review]
@kushal34712 This PR looks good to me.

File: README.md [hunk 1]
```diff
@@ -227,6 +227,11 @@ Volunteers have translated many of our Contributing, How-to, and Code of Conduct
 
 You might notice that there are [some missing translations here](docs/README.md#translations) - perhaps you would like to help out by [contributing a translation](docs/CONTRIBUTING.md#help-out-by-contributing-a-translation)?
 
+## Contributors
+
+<a href="https://github.com/EbookFoundation/free-programming-books/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=EbookFoundation/free-programming-books"  alt="free-programming-books Contributors"/>
```
Comment (sohansai) [inline README.md:233]: Adding your contribution would downplay the efforts of 2,600 others, so it's best not to include it; every contributor has made valuable contributions.

**Comment by kushal34712** [comment]
@honey-speaks-tech just accpet his one so that i can make those change and @sohansai i will other to see the contribution in the repo

File: README.md (hunk 1 above)
Comment (SethFalco) [reply README.md:233]: I'm not a big fan of this proposal tbh. The README is unfortunately quite bloated and uncomfortable to read already. ^-^'

Adding more images/widgets that don't actually help the reader or provide them with information they would want, would just further increase how much readers have to sift through before finding information relevant for them.

These widgets are great for relatively empty pages to pad them out, but not when we have more than enough content already.

**Comment by Thenlie** [comment]
I agree with @SethFalco and would close this.

**Pull request closed without merging**